src/status_im/ui/screens/add_new/new_chat/db.cljs

```diff
@@ -9,9 +9,12 @@
   [{:keys [multiaccount]} public-key]
   (= (:public-key multiaccount) public-key))
 
+(def empty-public-key "0x0400000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000")
```

Comment:
you can use this one https://github.com/status-im/status-react/blob/67f64f38168bd989b811a8f0e76d14dde79da6dd/src/status_im/ethereum/ens.cljs#L20

Reply:
Updated, thanks!